### PR TITLE
kubeadm: remove some unused function

### PR DESCRIPTION
**Pull request by my-git9**
#### What type of PR is this?

/kind cleanup

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
NONE
```

### CODE REVIEW

File: cmd/kubeadm/app/constants/constants.go [hunk 1]
```diff
@@ -577,16 +577,6 @@ func GetAdminKubeConfigPath() string {
...
-// GetSuperAdminKubeConfigPath returns the location on the disk where admin kubeconfig is located by default
-func GetSuperAdminKubeConfigPath() string {
-	return filepath.Join(KubernetesDir, SuperAdminKubeConfigFileName)
-}
-
-// GetBootstrapKubeletKubeConfigPath returns the location on the disk where bootstrap kubelet kubeconfig is located by default
-func GetBootstrapKubeletKubeConfigPath() string {
-	return filepath.Join(KubernetesDir, KubeletBootstrapKubeConfigFileName)
-}
-
```
Comment (neolit123) [inline cmd/kubeadm/app/constants/constants.go:589]: are there places where we should be calling these functions instead of constructing the paths there?

Comment (my-git9) [reply cmd/kubeadm/app/constants/constants.go:589]: Like these？
- https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/phases/kubeconfig/kubeconfig.go#L593
- https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/cmd/phases/join/kubelet.go#L116
- https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/cmd/phases/reset/cleanupnode.go#L164-L166

Comment (neolit123) [reply cmd/kubeadm/app/constants/constants.go:589]: seems like all places that construct the super-admin.conf path use a dynamic dir variable.

so the default path location  `filepath.Join(constants.KubernetesDir, constants.SuperAdminKubeConfigFileName)` is never used. it always like `filepath.Join(dirVariable, constants.SuperAdminKubeConfigFileName)`

```
$ grep SuperAdminKubeConfigFileName ./cmd/kubeadm/* -rnI | grep KubernetesDir
./cmd/kubeadm/app/phases/upgrade/staticpods.go:517:             if _, err := os.Stat(filepath.Join(pathMgr.KubernetesDir(), constants.SuperAdminKubeConfigFileName)); err == nil {
./cmd/kubeadm/app/constants/constants.go:585:   return filepath.Join(KubernetesDir, SuperAdminKubeConfigFileName)
./cmd/kubeadm/app/constants/constants_test.go:54:       expected := filepath.Join(KubernetesDir, SuperAdminKubeConfigFileName)
```

it seems fine to remove this function and tests.

/lgtm
/approve

**Review by neolit123 — COMMENTED** [review]
/priority backlog
/triage accepted

**Pull request merged**